package.json

```diff
@@ -19,8 +19,8 @@
     "lint-fix-js": "yarn lint-js --fix",
     "lint-css": "stylelint \"src/**/*.css\" \"res/**/*.css\"",
     "lint-fix-css": "yarn lint-css --fix",
-    "prettier-run": "prettier --check .",
-    "prettier-fix": "prettier --write .",
+    "prettier-run": "prettier --check . --cache --cache-strategy content --cache-location .prettiercache",
+    "prettier-fix": "prettier --write . --cache --cache-strategy content --cache-location .prettiercache",
```

Comment:
The default cache location is somewhere in node_modules, I was worried that it might get picked up by the cache happening in the CI. In CI I'd prefer to run without a cache, that's why I thought it would be better to explicitely set this cache location. Note that the eslint cache is in `.eslintcache` so it looked like a good idea to use a similar name.

Reply:
Good idea. I would also like to run the CI without a cache. Also it would be easy to remove the cache from the root directory if we want to, when we are developing locally.